### ISSUE TITLE
Broken on Windows

### ISSUE THREAD

**Issue by simon-p-r** [geek/find-rc#1]
Hey

This is output from running test on Windows, it is affecting lab upgrade for me due to this commit https://github.com/hapijs/lab/commit/4d228729a352a532be29c0a268c21a854610a53d

``` shell
D:\Modules\find-rc>npm t

> find-rc@3.0.0 pretest D:\Modules\find-rc
> npm run lint


> find-rc@3.0.0 lint D:\Modules\find-rc
> belly-button


Clean as a whistle! No issues found!


> find-rc@3.0.0 test D:\Modules\find-rc
> lab -c


<--- Last few GCs --->

   14567 ms: Mark-sweep 1379.3 (1434.9) -> 1379.2 (1434.9) MB, 248.4 / 0 ms [allocation failure] [GC in old space requested].
   14946 ms: Mark-sweep 1379.2 (1434.9) -> 1379.2 (1434.9) MB, 311.9 / 0 ms [allocation failure] [GC in old space requested].
   15254 ms: Mark-sweep 1379.2 (1434.9) -> 1379.2 (1434.9) MB, 308.0 / 0 ms [last resort gc].
   15563 ms: Mark-sweep 1379.2 (1434.9) -> 1379.2 (1434.9) MB, 308.9 / 0 ms [last resort gc].


<--- JS stacktrace --->

==== JS stack trace =========================================

Security context: 000001E8B85C9E79 <JS Object>
    2: normalize [path.js:~308] [pc=000001786FB2BEFE] (this=0000035349DFA5D1 <an Object with map 000003F9F430CBE1>,path=000002238A004101 <String[13]: D:\\.labrc.js>)
    3: join [path.js:~458] [pc=000001786FB44320] (this=0000035349DFA5D1 <an Object with map 000003F9F430CBE1>)
    4: arguments adaptor frame: 2->0
    5: dirPaths [D:\Modules\find-rc\node_modules\find-rc\lib\index.js:~44] [pc=00...

FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
npm ERR! Test failed.  See above for more details.
```

Thanks

**Comment by geek** [comment]
@simon-p-r this is now fixed in 3.0.1 and will be picked up when you install lab.

**Comment by simon-p-r** [comment]
Cheers @geek :+1:
